packages/ui/src/lib/components/icons/profile-default.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+	import type { IconProps } from '$lib/types'
+
+	type $$Props = IconProps
+
+	let cls: string | undefined = undefined
+	export { cls as class }
```

Comment:
So technically this should complain because the line `type $$Props = IconProps` dictates that the component should expose `size` prop as well. It does not complain now because we have outdated version of svelte but it will.... I'd suggest you add the `size` prop